snaps_k8s/k8s-deploy.yaml

```diff
@@ -0,0 +1,220 @@
+# Copyright 2018 ARICENT HOLDINGS LUXEMBOURG SARL and Cable Television
```

Comment:
Please move this example configuration down into the doc directory into another called like sample config

Reply:
Incorporated.